mlir/utils/jenkins/Dockerfile

```diff
@@ -124,7 +124,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 WORKDIR /MIOpenDepsWork
 RUN wget https://raw.githubusercontent.com/ROCmSoftwarePlatform/MIOpen/develop/requirements.txt \
     && wget https://raw.githubusercontent.com/ROCmSoftwarePlatform/MIOpen/develop/install_deps.cmake \
-    && sed -i -e '/llvm-project-mlir/d' requirements.txt \
+    && sed -i -e '/rocMLIR/d' requirements.txt \
     && cmake -P install_deps.cmake --prefix /usr/local
 RUN echo $(git ls-remote https://github.com/ROCmSoftwarePlatform/MIOpen HEAD) > rock-deps-commit-hash
```

Comment:
This one is missing: `miopen-deps-commit-hash`

Reply:
Yah, sounds good. Done in eafbbae